kitsune/wiki/views.py

```diff
@@ -1417,6 +1462,18 @@ def _document_form_initial(document):
             'needs_change_comment': document.needs_change_comment}
 
 
+def _draft_initial(type, draft_revision):
```

Comment:
This seems like it should be two methods, `_draft_initial_doc` and `_draft_initial_rev`.